.gitignore

```diff
@@ -65,7 +65,7 @@ playground.xcworkspace
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 Pods/
 # Add this line if you want to avoid checking in source code from the Xcode workspace
-*.xcworkspace
+# *.xcworkspace
```

Comment:
Revert?

Reply:
This was necessary for Bitrise since it need to path the workspace to build it for testing in its virtual environment.